src/core_modules/capture-core/components/Widget/widgetCollapsible.types.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import type { Node } from 'react';
+
+export type WidgetCollapsibleProps = {|
+    header: Node,
+    children: Node,
+    open: boolean,
+    onOpen: () => void,
+    onClose: () => void,
+    color?: ?string,
```

Comment:
Can we do `color?: string` instead. This way we don't allow `null` to be passed in and the benefit is that we can assign default values in the function declarations. For example in `WidgetNonCollapsible.component.js ` we can declare the function as this:
`const WidgetNonCollapsiblePlain = ({ header, children, color = colors.white, classes }: WidgetNonCollapsiblePropsPlain) =>`